zuul.d/projects.yaml

```diff
@@ -4,5 +4,15 @@
     github-check:
       jobs:
         - dataplane-operator-docs-preview
+        - openstack-k8s-operators-content-provider
+        - cifmw-data-plane-adoption-osp-17-to-extracted-crc:
+            dependencies:
+              - openstack-k8s-operators-content-provider
+            files:
+              - ^api/*
+              - ^config/*
+              - ^controllers/*
+              - ^pkg/*
```

Comment:
this is all of the code, so using files doesn't really help here. I don't think we want a 2.5 hour job on this repo.

Reply:
well there is also /docs /examples /hack ;)

can we further target this though? Looking under those directories it was not clear to me that we could.

We need to check if changes proposed to this operator will break our understanding of the adoption workflow (to this point in time), as codified in the adoption tests run in this job.

Reply:
> We need to check if changes proposed to this operator will break our understanding of the adoption workflow (to this point in time), as codified in the adoption tests run in this job.

I don't think that running it on every PR matches some of the earlier agreed upon context setting for which jobs we should run where. Perhaps we can revisit those discussions, but I'd like it to be more broadly understood so there's some acknowledgement about the impact of this change.

Reply:
the proposal is that it will not run on changes to /tests or /docs or the other directories.
as previously mentioned I didn't think we could make it more specific than I posted. However any coverage is better than no coverage so... can you propose an improvement to the files: list. Can/should we consider removing any of those? Can we be more specific with files and sub-directories? 

The idea is that we are constantly evolving our understanding of the adoption workflow, when new services are added (earlier on) or with new breaking features like the UpdateVersion resource and the latest work with OSPCIX-308. We have to update our "implementation" of the adoption workflow codified in the /tests/ subdirectory of the data-plane-adoption repo (and run in the adoption ci jobs).

It would be best to catch these breaking changes as close to the source as possible, which means gating on dataplane-operator and edpm-ansible.

We currently run adoption jobs in data-plane-adoption, ci-framework, install_yamls and openstack-operator repos. It seems to me that the omission of the actual dataplane related code is wrong.

Reply:
We've agreed to hold this PR for now, and then when dataplane-operator is merged into openstack-operator it will be covered by the adoption integration job. We can continue to refine with files/etc once that merge is complete.

Reply:
ok. for right now and as also agreed, we will proceed with the edpm-ansible so at least that is more than nothing :D https://github.com/openstack-k8s-operators/edpm-ansible/pull/662

Closing this pull.